src/locale/languages/en.js

```diff
@@ -157,6 +157,9 @@ export const en = {
     followingList: {
       title: 'Following',
     },
+    followYou: {
+      title: 'Follows you',
+    },
```

Comment:
As part of trying to streamline translation updates, could you add a comment to the top of `en.js` with the commit hash and link to the PR? That way we can sort of be able to tell if translations are not the same between multiple different languages.

CC @machour